### PR TITLE
fixbug:add membershipCard module and modify design

### PR DESCRIPTION
**Pull request by ShenDing1125**
- 新增會員方案模組 (Enterprise)

### CODE REVIEW

File: src/components/program/ProgramSubscriptionPlanAdminCard.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {
   PerpetualPlanModal,
   SubscriptionPlanModal,
 } from './programPlanAdminModals'
+import { GET_CARD_TITLE_BY_ID } from './programPlanAdminModals/formItem/MembershipCardItem'
```
Comment (TedxTed) [inline src/components/program/ProgramSubscriptionPlanAdminCard.tsx:25]: GetCardTitleById

File: src/components/program/ProgramSubscriptionPlanAdminCard.tsx [hunk 2]
```diff
@@ -66,6 +82,9 @@ const ProgramSubscriptionPlanAdminCard: React.FC<{
     appId,
     `ProgramPlan_${programPlan?.id}`,
   )
+  const { loading: loadingById, data: dataById } = useQuery(GET_CARD_TITLE_BY_ID, {
```
Comment (TedxTed) [inline src/components/program/ProgramSubscriptionPlanAdminCard.tsx:85]: GetCardTitleById

File: src/components/program/ProgramSubscriptionPlanAdminCard.tsx [hunk 3]
```diff
@@ -66,6 +82,9 @@ const ProgramSubscriptionPlanAdminCard: React.FC<{
...
+    variables: { id: programPlan.card_id },
```
Comment (TedxTed) [inline src/components/program/ProgramSubscriptionPlanAdminCard.tsx:86]: convert `cardId `

**Review by TedxTed — CHANGES_REQUESTED** [review]

**Review by TedxTed — APPROVED** [review]

File: src/components/program/ProgramSubscriptionPlanAdminCard.tsx (hunk 3 above)
Comment (kkshyu) [reply src/components/program/ProgramSubscriptionPlanAdminCard.tsx]: we can set the skip condition into `useQuery`
```ts
  const { loading: loadingById, data: dataById } = useQuery(GET_CARD_TITLE_BY_ID, {
    variables: { id: programPlan.card_id },
    skip: !!programPlan.card_id
```
FYI: https://www.apollographql.com/docs/react/api/react/hooks/#usequery-parameters-options-skip

**Pull request merged**